File: pset6/mario/mario.py
from cs50 import get_int

num = get_int("Height: ")
while (num > 8 or num < 1):
    num = get_int("Height: ")

n = num - (num - 1)
m = num - n

for i in range(num):

    # for j in range(m):
    #     print(" ", end = "")

    # for k in range(n):
    #     print("#", end = "")
    # # print()

    # for l in range(1):
    #     print("  ", end = "")
    for o in range(n):
        print("#", end = "")
    print()

    n+=1
    m-=1

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: \n");
    }
    while (height < 1 || height > 8);

    int n = height - (height - 1); //#
    int m = height - n; //spaces

    for (int i = 0; i < height; i++)
    {
        for (int k = 0; k < m; k++)
        {
            printf(" "); //aligns right
        }

        for (int j = 0; j < n; j++)
        {

            printf("#"); //prints left pyramid

        }

        for (int c = 0; c < 1; c++)
        {
            printf("  "); //prints space
        }

        for (int b = 0; b < n; b++)
        {
            printf("#");//prints right pyramid
        }

        printf("\n");
        n++;
        m = m - 1;



    }





}